examples/6field-simple/elm_6f.cxx

```diff
@@ -12,1677 +12,1682 @@
+    // nu_e.applyBoundary();
+    // mesh->communicate(nu_e);
 
-  B0vec.covariant = false;
-  B0vec.x = 0.;
-  B0vec.y = Bpxy / hthe;
-  B0vec.z = 0.;
+    if (diffusion_par > 0.0) {
 
-  // Set V0vec field vector
+      output.write("\tion thermal noramlized constant: Tipara1 = {:e}\n", Tipara1);
+      output.write("\telectron normalized thermal constant: Tepara1 = {:e}\n", Tepara1);
+      // xqx addition, begin
+      // Use Spitzer thermal conductivities
+      nu_i = 4.80e-8 * (Zi * Zi * Zi * Zi / sqrt(AA)) * LnLambda
+             * ((N0)*Nbar * density / 1.e6) * pow(Ti0 * Tibar, -1.5); // nu_i in 1/S.
+      // output.write("\tCoulomb Logarithm: {:e} \n", max(LnLambda));
+      output.write("\tion collision rate: {:e} -> {:e} [1/s]\n", min(nu_i), max(nu_i));
+
+      // nu_i.applyBoundary();
+      // mesh->communicate(nu_i);
+
+      vth_i = 9.79e3 * sqrt((Ti0)*Tibar / AA); // vth_i in m/S.
+      output.write("\tion thermal velocity: {:e} -> {:e} [m/s]\n", min(vth_i),
+                   max(vth_i));
+      // vth_i.applyBoundary();
+      // mesh->communicate(vth_i);
+      vth_e = 4.19e5 * sqrt((Te0)*Tebar); // vth_e in m/S.
+      output.write("\telectron thermal velocity: {:e} -> {:e} [m/s]\n", min(vth_e),
+                   max(vth_e));
+      // vth_e.applyBoundary();
+      // mesh->communicate(vth_e);
+    }
 
-  V0vec.covariant = false;
-  V0vec.x = 0.;
-  V0vec.y = Vp0 / hthe;
-  V0vec.z = Vt0 / Rxy;
+    if (compress0) {
+      eta_i0 = 0.96 * Pi0 * Tau_ie * nu_i * Tbar;
+      output.write("\tCoefficients of parallel viscocity: {:e} -> {:e} [kg/(m s)]\n",
+                   min(eta_i0), max(eta_i0));
+    }
 
-  // Set V0eff field vector
+    if (diffusion_par > 0.0) {
+      kappa_par_i = 3.9 * vth_i * vth_i / nu_i; // * 1.e4;
+      kappa_par_e = 3.2 * vth_e * vth_e / nu_e; // * 1.e4;
 
-  V0eff.covariant = false;
-  V0eff.x = 0.;
-  V0eff.y = -(Btxy / (B0 * B0)) * (Vp0 * Btxy - Vt0 * Bpxy) / hthe;
-  V0eff.z = (Bpxy / (B0 * B0)) * (Vp0 * Btxy - Vt0 * Bpxy) / Rxy;
+      output.write("\tion thermal conductivity: {:e} -> {:e} [m^2/s]\n", min(kappa_par_i),
+                   max(kappa_par_i));
+      output.write("\telectron thermal conductivity: {:e} -> {:e} [m^2/s]\n",
+                   min(kappa_par_e), max(kappa_par_e));
 
-  /**************** SET VARIABLE LOCATIONS *************/
+      output.write("\tnormalized ion thermal conductivity: {:e} -> {:e} \n",
+                   min(kappa_par_i * Tipara1), max(kappa_par_i * Tipara1));
+      output.write("\tnormalized electron thermal conductivity: {:e} -> {:e} \n",
+                   min(kappa_par_e * Tepara1), max(kappa_par_e * Tepara1));
 
-  P.setLocation(CELL_CENTRE);
-  U.setLocation(CELL_CENTRE);
-  phi.setLocation(CELL_CENTRE);
-  Psi.setLocation(CELL_YLOW);
-  if (emass)
-    Ajpar.setLocation(CELL_YLOW);
-  Jpar.setLocation(CELL_YLOW);
+      Field3D kappa_par_i_fl, kappa_par_e_fl;
```

Comment:
warning: multiple declarations in a single statement reduces readability [readability-isolate-declaration]

```cpp
Field3D kappa_par_i_fl, kappa_par_e_fl;
^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```